### PR TITLE
RavenDB-15603 Items disappear in databases view when it is portrait mode

### PR DESCRIPTION
**Pull request by kwiato**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/shell.ts [hunk 1]
```diff
@@ -290,6 +292,19 @@ class shell extends viewModelBase {
             this.mainMenu.update(items);
         };
 
+        
+        let checkScreenSize = () => {
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/shell.ts]: let -> const

File: src/Raven.Studio/wwwroot/App/views/shell.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div class="layout-container" data-bind="css: { 'collapse-menu': collapseMenu(), 'pin-notifications': notificationCenter.pinNotifications(), 'show-notifications': notificationCenter.showNotifications() }">
+<div class="layout-container" data-bind="css: { 'collapse-menu': collapseMenu() || smallScreen(), 'pin-notifications': notificationCenter.pinNotifications(), 'show-notifications': notificationCenter.showNotifications(), 'show-search': searchBox.showSearchSmallScreen() }">
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/shell.html:1]: I don't see declaration of: showSearchSmallScreen anywhere. Did you forgot to commit all files?

File: src/Raven.Studio/wwwroot/App/views/resources/databases.html [hunk 3]
```diff
@@ -104,110 +109,113 @@ <h2 class="text-center text-muted">No databases have been created</h2>
             </div>
         </div>
     </div>
-    <div class="flex-grow scroll" data-bind="if: databases().sortedDatabases().length, visible: databases().sortedDatabases().length">
+    <div class="flex-grow scroll" id="databaseList" data-bind="if: databases().sortedDatabases().length, visible: databases().sortedDatabases().length">
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/resources/databases.html]: do we need this id here? I don't see any usages.

File: src/Raven.Studio/wwwroot/App/views/shell.html (hunk 2 above)
Comment (kwiato) [reply src/Raven.Studio/wwwroot/App/views/shell.html:1]: I didn't stage it. Fixed it now.

**Review by ml054 — APPROVED** [review]

**Pull request merged**